src/resources/views/crud/buttons/delete.blade.php

```diff
@@ -1,6 +1,6 @@
 @if ($crud->hasAccess('delete'))
     <a href="javascript:void(0)" onclick="deleteEntry(this)" data-route="{{ url($crud->route.'/'.$entry->getKey()) }}" class="btn btn-sm btn-link" data-button-type="delete">
-        <span><i class="la la-trash"></i> {{ trans('backpack::crud.delete') }}</span>
+        <span><i class="la la-trash"></i> <span class="delete-button-text-span">{{ trans('backpack::crud.delete') }}</span></span>
```

Comment:
Do we really need two spans? One for the entire button content, one for the text? Cause if we only need one for the text, it should be as easy as moving the span start after the icon. No class needed:

```suggestion
        <i class="la la-trash"></i> <span>{{ trans('backpack::crud.delete') }}</span>
```

Reply:
![sma-sexiest-man-alive-gif-by-maytag](https://github.com/Laravel-Backpack/CRUD/assets/1032474/d5e72239-59d1-462b-afa7-6bae4883dd5b)